src/main/java/org/opensearch/ad/ml/CheckpointDao.java

```diff
@@ -425,6 +492,52 @@ private void logFailure(BulkByScrollResponse response, String detectorID) {
         }
     }
 
+    private RandomCutForest restoreRCFModel(String rcfCheckpoint) {
```

Comment:
Why is called restoreRCF instead of deserialize?

Reply:
changed to deserializeRCFModel.